src/pt/leitordemanga/build.gradle

```diff
@@ -0,0 +1,9 @@
+ext {
+    extName = 'Leitor de mangá'
```

Comment:
Why not `Leitor de Mangá`? The site uses that over the non-capitalized mangá.
If you change it, you should also change it in the `.kt` file.

Reply:
Resoved: https://github.com/keiyoushi/extensions-source/pull/2280/commits/d103e792cb35d0a7bc93c96cd83860cabf62c098